### PR TITLE
cs_instance: doc: fix typo in examples

### PR DESCRIPTION
**Pull request by resmo**
##### SUMMARY

fixes sanity

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

cs_instance

##### ANSIBLE VERSION

```paste below
2.7
```

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by gundalow** [comment]
Discusses `no` != `false` on IRC, I suggest adding to `ignore.txt` for the moment

File: lib/ansible/modules/cloud/cloudstack/cs_instance.py [hunk 1]
```diff
@@ -77,11 +77,12 @@
       - Name the hypervisor to be used for creating the new instance.
       - Relevant when using I(state=present), but only considered if not set on ISO/template.
       - If not set or found on ISO/template, first found hypervisor will be used.
-    choices: [ KVM, VMware, BareMetal, XenServer, LXC, HyperV, UCS, OVM, Simulator ]
+    choices: [KVM, kvm, VMware, vmware, BareMetal, baremetal, XenServer, xenserver, LXC, lxc, HyperV, hyperv, UCS, ucs, OVM, ovm, Simulator, simulator]
   keyboard:
     description:
       - Keyboard device type for the instance.
     choices: [ de, de-ch, es, fi, fr, fr-be, fr-ch, is, it, jp, nl-be, no, pt, uk, us ]
```
Comment (mattclay) [inline lib/ansible/modules/cloud/cloudstack/cs_instance.py]: Quote the `no` to avoid parsing as a bool.

Comment (resmo) [reply lib/ansible/modules/cloud/cloudstack/cs_instance.py]: *facepalm*

**Comment by mattclay** [comment]
The tests were unstable. I've restarted them. Assuming they pass, they may still warrant further investigation, as they're likely to be unstable in the future as well.

**Comment by resmo** [comment]
@mattclay I was able to reproduce the issue, it looks like a bug in cloudstack.

**Comment by gundalow** [comment]
Might be worth backporting this

**Comment by resmo** [comment]
preparing a couple of backports. Thanks!

**Pull request merged**